salvo-remote/azure-pipelines/bazel.yml

```diff
@@ -0,0 +1,73 @@
+parameters:
+- name: ciTarget
+  # The CI target to build, this is the $1 given to ci/do_ci.sh below.
+  displayName: "CI target"
+  type: string
+  default: "build"
+- name: repo
+  # The repository to build in.
+  displayName: "repository"
```

Comment:
Can we keep the first letter to be capital for all the disaplyName?

Reply:
Done, thank you.